index.html

```diff
@@ -16,12 +17,18 @@
     <section class="toolbar">
       <nav class="desknav">
         <div class="my-logo"><a class="my-logo" href="#">My Logo</a></div>
-        <div class="my-menu"><img src="img/hamburger.svg" alt="my-menu icon"></div>
         <div class="items">
           <div><a href="#portfolio">Portfolio</a></div>
           <div><a href="#about">About</a></div>
           <div><a href="#contact">Contact</a></div>
         </div>
+        <div class="my-menu">
+          <div class="item close"><a href="#" class="c-m"><i class="fa fa-times " aria-hidden="true"></i></a></div>
+          <div class="mobile-nav"><a href="#portfolio" class="c-m">Portfolio</a></div>
+          <div class="mobile-nav"><a href="#about" class="c-m">About</a></div>
+          <div class="mobile-nav"><a href="#contact" class="c-m">Contact</a></div>
+        </div>
+        <div class="item menu"><a href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></a></div>
```

Comment:
- [ ] it would be better if you add  the blur effect as in the Figma design in order to match the design 
- Yours 
<img width="323" alt="Screenshot 2023-01-02 at 14 07 58" src="https://user-images.githubusercontent.com/61790587/210235844-5f1db29c-1d81-43f0-8ea8-b7465b621564.png">
- Design 
<img width="275" alt="Screenshot 2023-01-02 at 14 08 53" src="https://user-images.githubusercontent.com/61790587/210235853-68fda76f-393f-478d-a7a4-c45ab69c57d6.png">